docs/rules/newline-after-var.md

```diff
@@ -23,107 +23,115 @@ This rule enforces a coding style where empty newlines are required or disallowe
 
 ## Options
 
-This rule takes one option, a string, which can be:
+This rule has a string option:
 
-* `"always"` enforces empty newlines after `var`, `let` or `const` (default)
-* `"never"` disallows empty newlines after `var`, `let` or `const`
+* `"always"` (default) requires an empty newline after `var`, `let`, or `const`
+* `"never"` disallows empty newlines after `var`, `let`, or `const`
 
-The following patterns are considered problems:
+### always
+
+Examples of **incorrect** code for this rule with the default `"always"` option:
 
 ```js
 /*eslint newline-after-var: ["error", "always"]*/
+/*eslint-env es6*/
 
 var greet = "hello,",
     name = "world";
 console.log(greet, name);
-```
-
-```js
-/*eslint newline-after-var: ["error", "never"]*/
-/*eslint-env es6*/
 
 let greet = "hello,",
     name = "world";
 
 console.log(greet, name);
-```
-
-```js
-/*eslint newline-after-var: "error"*/  // defaults to always
-/*eslint-env es6*/
 
 var greet = "hello,";
 const NAME = "world";
 console.log(greet, NAME);
+
+var greet = "hello,";
+var name = "world";
+// var name = require("world");
+console.log(greet, name);
 ```
 
-The following patterns are not considered problems:
+Examples of **correct** code for this rule with the default `"always"` option:
 
 ```js
 /*eslint newline-after-var: ["error", "always"]*/
+/*eslint-env es6*/
 
 var greet = "hello,",
     name = "world";
 
 console.log(greet, name);
-```
-
-```js
-/*eslint newline-after-var: ["error", "never"]*/
-/*eslint-env es6*/
 
 let greet = "hello,",
     name = "world";
 console.log(greet, name);
-```
-
-```js
-/*eslint newline-after-var: "error"*/  // defaults to always
-/*eslint-env es6*/
 
 var greet = "hello,";
 const NAME = "world";
 
 console.log(greet, NAME);
+
+var greet = "hello,";
+var name = "world";
+// var name = require("world");
+
+console.log(greet, name);
 ```
 
-Note: in `"always"` mode, comments on a line directly after var statements are treated like additional var statements.
```

Comment:
Could a simpler version of this thought go with the description of options?

* `"always"` (default) requires an empty newline after `var`, `let`, or `const`

    Comments on a line and so on [with 4-space indent and separated by empty lines, haha, this has the same indent as surrounding bulleted list items, like List Continue style in a word processor]

* `"never"` disallows empty newlines after `var`, `let`, or `const`